hstream-sql/src/HStream/SQL/Extra.hs

```diff
@@ -26,14 +35,14 @@ extractPNDouble (IPDouble _ n) = n
 extractPNDouble (NDouble  _ n) = (-n)
 
 extractColumnIdent :: ColumnIdent -> Text
-extractColumnIdent (ColumnIdentNormal _ (Ident text)) = text
-extractColumnIdent (ColumnIdentRaw _ (QuotedRaw text')) =
-  Text.tail . Text.init $ text'
+extractColumnIdent (ColumnIdentNormal _ (Ident text)) = Text.toLower text
+extractColumnIdent (ColumnIdentDoubleQuoted _ (DoubleQuoted text)) =
+  Text.tail . Text.init $ text
 
 extractHIdent :: HIdent -> Text
-extractHIdent (HIdentNormal _ (Ident text)) = text
-extractHIdent (HIdentRaw _ (QuotedRaw text')) =
-  Text.tail . Text.init $ text'
+extractHIdent (HIdentNormal _ (Ident text)) = Text.toLower text
+extractHIdent (HIdentDoubleQuoted _ (DoubleQuoted text)) =
+  Text.tail . Text.init $ text
```

Comment:
There are some generated identifiers (e.g. sink stream from a push query) that contain capitals. As a consequence, some of them can not be found by SQL. (e.g. `SELECT ... EMIT CHANGES` from cli)

Reply:
Yes. But do consider all of the corner cases. For example, by executing a query such as 

```
SELECT * FROM s1 EMIT CHANGES;
```

A temporary stream is auto-generated, say `cli_generated_stream_pjrRHbqqiSCwlvbMjRsD`, see `Client/Internal.hs#genRandomSinkStreamSQL`. The stream name is not double-quoted there, which causes a simple query to fail because `cli_generated_stream_pjrRHbqqiSCwlvbMjRsD` and `cli_generated_stream_pjrrhbqqiscwlvbmjrsd` are just different objects.

Reply:
thx, Seems that we should figure out the places where the names are auto generated and where the tests use string concat and needs to fix. Temp revert [here](https://github.com/hstreamdb/hstream/pull/1454/commits/8606659ca1c66aaf679e647f719b90041bcded2f).